csharp/Sdk.Examples/Horizon/Tutorials/Instrument/Instrument.cs

```diff
@@ -0,0 +1,57 @@
+using System.Threading.Tasks;
+using Finbourne.Horizon.Sdk.Model;
+using NUnit.Framework;
+using Sdk.Examples.Horizon.Utilities;
+
+namespace Sdk.Examples.Horizon.Tutorials.Instrument
+{
+    [TestFixture]
+    public class Instrument : TutorialBase
+    {
+        [Test]
+        public async Task Search_OpenFigi_Query()
```

Comment:
this is technically a LUSID API and probably doesnt need to exist here as a standalone example